### PR TITLE
Feature/ios page 175 

### PR DESCRIPTION
**Pull request by ckoutsiaris**
https://github.wdf.sap.corp/gd/FioriExperience/issues/175

Because this one is getting too big I thought to split it in smaller PR's. 

In this PR the following:

1. Created Page for iOS
2. Extended the LeftNav to have an initial state closed if we have this requirement
3. Since ResourcesList and LearningList are components I moved them under the components folder
4. Extended the ResourcesList so that we can be filtered by category
5. Created a basic tabs component. Tab values are coming from frontmatter

So far:
screenshot_2019-02-27 sap fiori

I will do some cleanup on the css afterwards.

### CODE REVIEW

File: src/pages/guideline_ios/index.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+            css={{
+              width: '100%'
+            }}
+          >
+            <div
+              css={{
+                width: '100%',
+                backgroundColor: '#f8f9fb',
+                height: '682px',
+                backgroundImage: 'url(' + designImg + ')',
```
Comment (eouin) [inline src/pages/guideline_ios/index.js]: @ckoutsiaris I reckon on a large screen the hero banner should still be centre align to the grid, not right aligned.

Comment (ckoutsiaris) [reply src/pages/guideline_ios/index.js]: I copy-pasted from existing. Will change it thanks

File: src/components/ResourceList/ResourcesList.js [hunk 2]
```diff
@@ -73,49 +106,30 @@ class ResourcesList extends React.Component {
...
-                      }}
-                      key={idx}
-                      data={entry}
-                    />
-                  ))}
-                </div>
-              )
-            )}
+            {
+              this.props.resource ?
```
Comment (MerlindlH) [inline src/components/ResourceList/ResourcesList.js]: I think you could pull this condition into the map

File: src/pages/guideline_ios/index.js [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+              font-size: 36px;
+              font-weight: normal;
+              height: 43px;
+              letter-spacing: 0.11px;
+              line-height: 43px;`}>explore Fiori for iOS</h1>
+              <Tabs>
+                {frontmatter.tabs.map((tab, idx) => {
+                  return (
+                    <div label={tab} key={idx}>
+                      Hello from {tab}!
```
Comment (MerlindlH) [inline src/pages/guideline_ios/index.js]: is this final?

Comment (ckoutsiaris) [reply src/pages/guideline_ios/index.js]: nope, we are going to have some cards there

File: src/components/ResourceList/ResourcesList.js (hunk 2 above)
Comment (ckoutsiaris) [reply src/components/ResourceList/ResourcesList.js]: correct, will do

**Comment by ckoutsiaris** [comment]
Thanks!

File: src/components/ResourceList/ResourcesList.js (hunk 2 above)
Comment (ckoutsiaris) [reply src/components/ResourceList/ResourcesList.js]: Actually, you can't break in array.map

**Pull request merged**